### DISCUSSION TITLE
Pause button is still clickable when pause menu appeared

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#24023] [Q&A] [answered]
https://youtu.be/U-rSzMsiSc4  
as you can see I still can click at pause button but it will not do anything though and also the confirmation progress bar is also out of place

**Comment by JacksonChen666** [comment, marked as answer]
are you confusing the [right click indicator](https://www.youtube.com/watch?v=GCXaQeFlaBY&t=272) with the pause button being "clickable"?

**Reply by timelessnesses** [reply]
I didn't know that's a right click indicator sorry 😅
